src/share/cxx/vector/KokkosKernels_Vector_SIMD.hpp

```diff
@@ -42,6 +42,25 @@
 #ifndef __KOKKOSKERNELS_VECTOR_SIMD_HPP__
 #define __KOKKOSKERNELS_VECTOR_SIMD_HPP__
 
+#if defined(__INTEL_COMPILER)
+
+#define NO_VECTOR_DEPS _Pragma("simd")
```

Comment:
Probably should be SIMD rather than NO_VECTOR_DEPS. The second sounds more like pragma ivdep, which is not as strong as pragma simd. pragma simd is roughly ivdep + always-vectorize.